app/views/admin/announcements/index.html.slim

```diff
@@ -0,0 +1,10 @@
+div.page-header
+  h1
+    | Instance Announcements
+    - if can?(:manage, Instance::Announcement)
+      div.pull-right
+        = link_to(t('.new'),
+          new_admin_announcement_path,
+          class: ['btn', 'btn-primary'])
+- @announcements.each do |announcement|
```

Comment:
I think <code>render @announcements</code> will do ? Or you prefer this for some reason ?